docs/success-guide/setup_config_files.md

```diff
@@ -159,6 +160,7 @@ Note that:
 - Accounts that would have been removed or updated but were not because of this feature are listed as `info` level log entries.
```

Comment:
Actually, they are listed as `debug` level log entries, not `info`.  Only the count of these users is at info level.